api/openapi.yaml

```diff
@@ -1612,6 +1411,12 @@ components:
           # on the client side.
           #
           # readOnly: true
+        terminated:
```

Comment:
this is for all assets (e.g. also offline assets like directories). Maybe we should rename it (also with a comment), or move into the asset info?

Reply:
Yes this is for all assets, it not specific to VMs, any needs to be marked as removed at somepoint, i.e. the directory gets deleted, the container is rm'd, the container image is removed from the registry. Best word I could think of was "terminated" to represent this, perhaps "deleted" would be better.

Reply:
nm :)